.rubocop.yml

```diff
@@ -13,6 +12,18 @@ Metrics/AbcSize:
 Bundler/DuplicatedGem:
   Enabled: false
 
+Style/Send:
```

Comment:
Ones we remove all the `rubocop_todo` in all gems we compare them and add this in the global `rubocop.yml` in meta